src/scores.jl

```diff
@@ -584,3 +584,34 @@ function num_cut_edges(name::String)::PlanScore
     end
     return PlanScore(name, score_fn)
 end
+
+
+function coerce_dist_agg_to_be_float!(graph::BaseGraph,
```

Comment:
I would love for this function to be more general - perhaps `coerce_attribute_type(graph::BaseGraph, key::String, new_type::DataType)`. Then `coerce_types_on_graph` can then call `coerce_attribute_type(graph, score.key, Float64)` for each `DistrictAggregate`. 

By making this function more general, users can also easily pre-process the graph so that the `DistrictScore`s/`PlanScore`s will work with the right data type. We can edit the starter code + documentation to reflect this. What do you think?